### PR TITLE
Update documentation for blog post

### PR DESCRIPTION
**Pull request by danvk**
- Link off to the demo
- Add some usage documentation (both JS & CSS)

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/pileup.js/198)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +19,62 @@ To play with the demo, you'll need to install the node.js [http-server][hs]:
...
+For example, to allocate 1/3 of the space to a variant track and 2/3 to a
+pileup track, you could use this CSS:
+
+```css
+.track.variants { flex: 1; }
+.track.pileup   { flex: 2; }
+```
+
+To style multiple tracks of the same type, you can use the `cssClass` property.
+
```
Comment (armish) [inline README.md]: people will love this flexibility! Really cool.

File: README.md [hunk 2]
```diff
@@ -27,8 +85,6 @@ Once installed, start sniper in the `pileup.js` folder:
 
 And browse to [http://localhost:9090/examples/playground](http://localhost:9090/examples/playground).
 
-![pileup.js screenshot](./pileup-screenshot.png)
```
Comment (armish) [inline README.md]: The screenshot looks much better at the top :+1:

**Comment by armish** [comment]
looks great @danvk. Do you think we should also put the license information at the bottom of the README? `package.json` and `LICENSE` files are already there for this, but might be good to have that on the GitHub page as well.

**Comment by danvk** [comment]
Sure, I added a link at the bottom of `README.md`.

**Comment by armish** [comment]
looking forward to the blog post :eyes:

**Pull request merged**